flask/app/participants.py

```diff
@@ -237,6 +243,7 @@ def create_participant():
         affected=request.json.get("affected"),
         solved=request.json.get("solved"),
         participant_type=request.json.get("participant_type"),
+        institution_id=request.json.get("institution_id"),
```

Comment:
Can you make this endpoint accept the text form of institution instead and do the same for `/api/_bulk`?

Reply:
revised in 4f944d550680490e9aded34026c3e4d37721f918